Solutions/STM32F4DISCOVERY/STM32F4DISCOVERY.settings

```diff
@@ -16,6 +16,7 @@
     <IsSolutionWizardVisible>True</IsSolutionWizardVisible>
     <ENDIANNESS>le</ENDIANNESS>
     <NO_BOOTLOADER_COMPRESSION>true</NO_BOOTLOADER_COMPRESSION>
+    <PLATFORM_EMULATED_FLOATINGPOINT Condition="'$(COMPILER_TOOL)'=='GCC'">true</PLATFORM_EMULATED_FLOATINGPOINT>

```

Comment:
Ah, I have found the issues, thanks.

So, if I understand it correctly, this happens only when crypto.lib is used (resp. installed)? Because IMHO it would be much better to fix the cause in crypto.lib (as you suggest in the other comment) than modifying floating point setting (which would be needed for all solutions configured to use newlib-nano).